src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -107,9 +107,14 @@ public Person toModelType() throws IllegalValueException {
             personTags.add(tag.toModelType());
         }
 
+
         final List<Exam> personExams = new ArrayList<>();
         for (JsonAdaptedExam exam : exams) {
-            personExams.add(exam.toModelType());
+            if (!exam.isExamOverdue(exam)) {
+                personExams.add(exam.toModelType());
+            } else {
+                System.out.println("removed past exams: " + exam);
```

Comment:
nice changes, but i think can remove print statements

Reply:
I added for us to keep track of the process. ok will remove